### PR TITLE
#255 Draft description of block

### PR DESCRIPTION
**Pull request by lucafavatella**
Please refer to commit message for details.

### CODE REVIEW

File: docs/design/blocks.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+It includes:
+* Block header;
+* Transactions;
+* Partial representation of state Merkle trees (e.g. accounts,
+  channels) before application of transactions;
+  * Each of such state Merkle tree is represented as:
+    * Proof and datum of each leaf affected by at least one
+      transaction in the block;
+    * Hash of the root.
+* Hash of the root of each of the state Merkle trees (e.g. accounts,
```
Comment (zbigniewpekala) [inline docs/design/blocks.md]: Hash of previous block? Height?

File: docs/design/blocks.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Block
+
+The following is a simplified model of the data structures related to
+the block.
+
+## Block
+
+It includes:
+* Block header;
```
Comment (zp-sd) [inline docs/design/blocks.md]: By block header do you mean everything listed in `## Block header`? Maybe we can have a reference here?

Comment (lucafavatella) [inline docs/design/blocks.md]: Yes, that is what I mean. Let me see how to make ref.

**Comment by lucafavatella** [comment]
> Maybe we can have a reference here?

Comment by @zp-sd addressed in 07e9642.

**Comment by sennui** [comment]
I am handing this of to @zp-sd to introduce more detail

**Pull request merged**